conf/modules.config

```diff
@@ -35,99 +35,188 @@ process {
         publishDir = [
             path: { "${params.outdir}/reference/bwa" },
             enabled: "params.save_reference",
-            pattern: "*{amb,ann,bwt,pac,sa}"
+            pattern: "*{amb,ann,bwt,pac,sa}",
+            mode: 'copy'
```

Comment:
Is this needed, because we have https://github.com/nf-core/sarek/blob/b389b1e3c679e9353b46af4d131e2961a2752cf0/conf/modules.config#L18

Reply:
https://github.com/nf-core/sarek/issues/471 see here, nick has also worked on this. i was testing something and thought this was the issue (was a different problem in the end)

Reply:
I see, I did not remember that...